src/commands/games/blackjack.ts

```diff
@@ -186,10 +186,21 @@ export class BlackjackCommand extends Command {
   async messageRun(message: Message, args: Args): Promise<Message> {
     // if there are no arguments, then resolve to the default bet amount; if there is only one argument and it is an
     // integer, then this is the bet amount; otherwise, reply that a valid bet amount must be entered
-    const bet = args.finished
+    const betAmount = await args.restResult<number>('integer');
+    const betPercent = await args.restResult<string>('string');
+    let bet = args.finished
```

Comment:
where's the please enter valid bet amt?


screenshots of slash and dot command working?
suppose i did .bj -50% or 200% or 0.001%
i don't do QA

Reply:
I didn't mess with the bet amount validator, so in order what happens is:
- gets percentage
- gets user's coins
- process percentage
- assign it back to the `bet` variable

and the rest of the process would go as planned